Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteScrollingCoordinator.h

```diff
@@ -96,6 +96,7 @@ class RemoteScrollingCoordinator final : public WebCore::AsyncScrollingCoordinat
     void receivedWheelEventWithPhases(WebCore::PlatformWheelEventPhase phase, WebCore::PlatformWheelEventPhase momentumPhase);
     void startDeferringScrollingTestCompletionForNode(WebCore::ScrollingNodeID, WebCore::WheelEventTestMonitor::DeferReason);
     void stopDeferringScrollingTestCompletionForNode(WebCore::ScrollingNodeID, WebCore::WheelEventTestMonitor::DeferReason);
+    void scrollingTreeNodeScrollbarVisibilityDidChange(uint64_t, WebCore::ScrollbarOrientation, bool);
```

Comment:
This should use `WebCore::ScrollingNodeID`, not `uint64_t`

Reply:
Fixed.